frontend/js/restqueries.js

```diff
@@ -0,0 +1,82 @@
+try {
+	var staticURL = "http://ec2-34-208-19-115.us-west-2.compute.amazonaws.com:18080/character/";
+	var playerURL = "OtherTestChar";
+	var DnDObj = getUser();
+	//getJSONField("player");
+}
+catch (err) {
+	console.log(err);
+}
+//getJSONField("name");
+populateDNDFields();
+
+function getUserText() {
+	var xhttp = new XMLHttpRequest();
+	xhttp.open("GET", staticURL + playerURL, false); //find a way to set async to true and have it return when it has something
```

Comment:
We definitely want to change this to be async in the future, but don't worry about that for now.